MaterialComponent.js

```diff
@@ -53,23 +57,34 @@ export default class MaterialComponent extends Component {
     }
     return classText;
   }
+
   // Components must implement this method for their specific DOM structure
   materialDom(props) {
     return h("div", Object.assign({}, props), props.children);
   }
+
   render() {
     this.buildClassName();
     // Fetch a VNode
     const componentProps = this.props;
-    if (componentProps.class) {
-      // We delete class prop here so that any sub node's class doesn't get over-ridden from this
-      delete componentProps.class;
-    }
+    const userDefinedClasses =
+      componentProps.className || componentProps.class || "";
```

Comment:
Change it to `((componentProps.className || "") + " " + (componentProps.class || ""))` to allow for having both `className` and `class`.

Reply:
both at the same time! ppl do that?

Reply:
Not sure why they would do that. I only use `class` but people are stupid.